.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Benchmarks
+on:
+  schedule:
+    - cron: "0 0 * * 0-6"  # Run daily at 0am

```

Comment:
To discuss with @mlxd - we were thinking that running on every commit to `master` might be worthwhile here.

Reply:
@mlxd, this comment is waiting for your feedback. Let us know if you have something in mind.